job-engine/client/src/main/java/org/eclipse/kapua/job/engine/client/queue/JobEngineQueueClientModule.java

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
```

Comment:
This and following factories were needed to provide concrete implementation of serializable interfaces for those modules referencing kapua-job-engine-client